### PR TITLE
Sample fastly block by coutry code and user agent

### PR DESCRIPTION
**Pull request by michaeldyl520**
## Purpose of this pull request

This pull request (PR) ...

## Affected DevDocs pages
https://devdocs.magento.com/guides/v2.3/cloud/cdn/fastly-vcl-blacklist.html

## Links to Magento source code
https://github.com/magento/devdocs/blob/master/src/guides/v2.3/cloud/cdn/fastly-vcl-blacklist.md

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.2/cloud/cdn/fastly-vcl-blacklist.md [hunk 1]
```diff
@@ -50,6 +50,28 @@ Create an `blocklist.json` file with the following JSON content:
 }
 ```
 
+## Sample Block By Country Code (A two-character ISO 3166-1 country code for the country associated with the IP address.)
```
Comment (dshevtsov) [inline src/guides/v2.2/cloud/cdn/fastly-vcl-blacklist.md]:
```suggestion
**Sample Block By Country Code (A two-character ISO 3166-1 country code for the country associated with the IP address.):**

```

File: src/guides/v2.2/cloud/cdn/fastly-vcl-blacklist.md [hunk 2]
```diff
@@ -50,6 +50,28 @@ Create an `blocklist.json` file with the following JSON content:
...
+{
+  "name": "blockbycountrycode",
+  "dynamic": "0",
+  "type": "recv",
+  "priority": "5",
+  "content": "if ( geoip.country_code == \"HK\" ) { error 405 \"Not allowed\";}"
+}
+```
+
+## Sample Block By http.User-Agent
```
Comment (dshevtsov) [inline src/guides/v2.2/cloud/cdn/fastly-vcl-blacklist.md]:
```suggestion
**Sample Block By http.User-Agent:**

```

**Comment by meker12** [comment]
running tests

File: src/guides/v2.2/cloud/cdn/fastly-vcl-blacklist.md (hunk 2 above)
Comment (meker12) [reply src/guides/v2.2/cloud/cdn/fastly-vcl-blacklist.md]: Will make these suggested changes in a separate PR.

File: src/guides/v2.2/cloud/cdn/fastly-vcl-blacklist.md (hunk 1 above)
Comment (michaeldyl520) [reply src/guides/v2.2/cloud/cdn/fastly-vcl-blacklist.md]: There is a need to ban access through the country code.

**Pull request merged**